### ISSUE TITLE
Enabling SpellCheck causes QupZilla to freeze

### ISSUE THREAD

**Issue by JHooverman** [QupZilla/qupzilla#1775]
Please try the following:

(1) Go to `Preferences -> Other-> SpellCheck settings`
(2) Click on `SpellCheck`

Result: QupZilla freezes and has to be killed.

Tested with:
Application version: 1.8.9
Qt version: 5.5.1
WebKit version: 538.1
Platform: Windows 7

**Comment by nowrep** [comment]
What about opening the dialog from webmenu (right click on some input on page)?

**Comment by JHooverman** [comment]
This is exactly what I have tried, but I did not find a spellcheck menu item:


![2015-11-19 14_05_21-enabling spellcheck causes qupzilla to freeze issue 1775 qupzilla_qupzilla](https://cloud.githubusercontent.com/assets/733365/11272078/a57ac3f4-8eca-11e5-8fe6-86f1829e2936.png)


So, I thought, I had to enable Spellcheck in the `Preferences`, which caused the issue I have described. Do I miss something?

**Comment by nowrep** [comment]
That may be the issue, if the spellcheck is not loaded.

**Comment by JHooverman** [comment]
I tried it with `qupzilla:config` and here I do not have a spellcheck entry in the context menu as well. The `qupzilla:home` can not be loaded (Error Message: `Server not found. QupZilla can't load page.`)

**Comment by nowrep** [comment]
Sorry, it is `qupzilla:start` :D

**Comment by smarquespt** [comment]
And maybe this one is also related:

Go to settings -> Others -> Click on Spellcheck button.

Result:

Spelcheck dialog is hidden below main preferences dialog.

Expected - Should be in front

**Comment by nowrep** [comment]
@smarquespt That's different, bug with modality.

**Comment by smarquespt** [comment]
@nowrep 

Should I open a new issue?

**Comment by nowrep** [comment]
Yes, please.

**Comment by smarquespt** [comment]
Done - https://github.com/QupZilla/qupzilla/issues/1777

Thanks

**Comment by JHooverman** [comment]
Closing the issue in favor of #1777.

**Comment by nowrep** [comment]
It is the same issue? So this is not actually freeze (crash)?

**Comment by JHooverman** [comment]
Yes, this is the same issue as #1777. After looking into it I have noticed, that the `Preferences`-dialog has not the color of an active window. So, I have moved this dialog to the left and tried to click on the `SpellCheck`-button and got this:

![2015-11-20 10_18_42-issues qupzilla_qupzilla - qupzilla](https://cloud.githubusercontent.com/assets/733365/11296133/8faaea6c-8f70-11e5-9988-e91f621fc3a4.png)

As @smarquespt stated correctly, the `SpellCheck`-dialog opens behind the `Preferences`-dialog.
